server/routes/logic.js

```diff
@@ -0,0 +1,123 @@
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
+const db = require('../db/db'); // ../db/db is based on how far from index.js
+const jwt = require('jsonwebtoken');
+const cookie = require("cookie");
+
+// Check if the user is already authenticated
+const routeAccessibility = async (req, res, next) => {
+  const authCookie = req.cookies.AccessToken;
+  if (authCookie) {
+    jwt.verify(authCookie, process.env.ACCESS_TOKEN_SECRET, (err, user) => { // user is a value serialised wich is authenticatedUser
+      if (err) return res.sendStatus(403);
```

Comment:
Indentation is weird here.  Also, let's log this error

Reply:
Would you care to explain bit more details?
Log equipped.